app/post.go

```diff
@@ -809,6 +810,10 @@ func (a *App) GetPostsPage(options model.GetPostsOptions) (*model.PostList, *mod
 		}
 	}
 
+	if appErr := a.filterInaccessiblePosts(postList); appErr != nil {
+		return nil, model.NewAppError("GetPostsPage", "app.post.filter_inaccessible_posts.app_error", nil, appErr.Error(), http.StatusInternalServerError)
```

Comment:
Return the error directly. No need to unwrap-wrap again. Same applies for other instances in this file.